langtest/modelhandler/transformers_modelhandler.py

```diff
@@ -707,8 +704,8 @@ def load_model(cls, path: str):
         if isinstance(path, str):
```

Comment:
whenever the **path** is not a string in `load_model()`, then **tokenizer** is None